### ISSUE TITLE
Clicking only works on the center of Drag handlers

### ISSUE THREAD

**Issue by YahngSungho** [atlassian/react-beautiful-dnd#2417]
![beautifulDnDClick](https://user-images.githubusercontent.com/29212791/174426160-fef56ee1-7cee-45b7-9df6-ed2fab1167b4.gif)

As shown in the attached GIF, clicks only work on the center of the elements. I want clicks to also work on the edges of drag handlers...

In this case, what keywords should I search for to solve the problem?

My code is as follows (simplified)

```
    <div {...dragableHandleProps}>
        <button />
    </div>
```

And here is the real code

```
    <div {...dragableHandleProps}>
        <Button icon={'drag-handle-vertical'} minimal className={'visibleIfHover3 grabCursor'} />
      </div>
```

The Button component is from https://blueprintjs.com/docs/#core/components/button
But anyway, the click event is captured by the div tag

**Comment by YahngSungho** [comment]
Does anyone have any clues? Please...

**Comment by yobananaboy15** [comment]
I managed to solve it by removing the button component and simply wrapping the Icon in a div. Try that.

**Comment by YahngSungho** [comment]
@yobananaboy15 Do you mean that you replaced the button with the icon??

**Comment by yobananaboy15** [comment]
Exactly. Try removing the button and replace it with the icon you want to use.
